assets/src/app/layout/main/layout.tsx

```diff
@@ -1,11 +1,9 @@
 import { observer } from 'mobx-react-lite';
-import { Toaster } from 'react-hot-toast';
 import { Header } from '~/components/header';
 
 export const MainLayout = observer(({ children }: { children: any }) => {
   return (
     <div className='min-h-screen'>
-      <Toaster />
```

Comment:
Huh?

Reply:
We have 2 toaster providers. main in app.tsx please check it